packages/admin-panel/src/pages/resources/DashboardItemsPage.js

```diff
@@ -42,6 +44,12 @@ const IMPORT_CONFIG = {
   },
 };
 
+const renderNewDashboardVizButton = () => (
+  <LightOutlinedButton startIcon={<AddCircleIcon />} href="/viz-builder/dashboard-item/">
```

Comment:
Another note is that previously this used to take us to the `/viz-builder/new/` endpoint, which prompted the user to enter the dashboard metadata details (code, name, permission group). I think keeping that flow is valuable, as we've also got designs for the UI to have buttons for templates and the like there.

Since map overlays have different metadata details, we may need a separate page for them, eg. `/viz-builder/dashboard-item/new` and `/viz-builder/map-overlay/new`

Reply:
>Not entirely sure if this is the cause, but the session management with the new New button flow seems a little differently to previously. If you were in the viz-builder and hit the Admin Panel icon in the top left, you'd be taken to the Admin Panel still logged in (likewise if you hit back in your browser). Now it seems to lose your session and take you back to the login page.